bundles/org.openhab.binding.bticinosmarther/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -26,41 +26,54 @@
 		<!-- Parameters -->
 		<parameter name="subscriptionKey" groupName="subscription" type="text" pattern="[0-9a-f]{32}">
 			<label>Subscription Key</label>
-			<description>This is the Subscription Key provided by BTicino/Legrand when you subscribe to Smarther - v2.0 product. Go to
-https://developer.legrand.com/tutorials/getting-started/</description>
+			<description>This is the Subscription Key provided by BTicino/Legrand when you subscribe to Smarther - v2.0 product.
+				Go to
+				https://developer.legrand.com/tutorials/getting-started/</description>
```

Comment:
the formatting for the descriptions in this file probably need to be manually adjusted.

Reply:
Manually adjusting the formatting is exactly what we want to avoid with the spotless formatter, or am I wrong there?